Convert helpers/general and storage.js to ts

johnschoeman opened the pull request:
Why:
We would like to use typescript on some files which import theses
functions and constants.

This commit:
Updates the files to ts.  Some rather permissive types are being
returned from the functions, which could likely be tightened up in a
future commit.

johnschoeman commented:
trying to ship smaller commits

JacobJaffe commented on app/helpers/General.ts:
```diff
@@ -1,34 +1,30 @@
 
-    return JSON.parse(data);
+    if (data) {
+      return JSON.parse(data);
```
Doesn't this return an any type? 
Without explicit setters / getters per payload to the store I think this whole `GetStoreData` needs to be typed as an `any` return

johnschoeman replied:
you are correct! good catch

JacobJaffe commented on app/TracingStrategyAssets.ts:
```diff
@@ -0,0 +1,103 @@
+import { useTranslation } from 'react-i18next';
```
It looks like to me this file got added without removing the corresponding .js version?

johnschoeman committed:
Convert helpers/general and storage.js to ts

Why:
We would like to use typescript on some files which import theses
functions and constants.

This commit:
Updates the files to ts. Some rather permissive types are being
returned from the functions, which could likely be tightened up in a
future commit.

tparesi reviewed (approved):
LGTM
